store/src/main/java/com/nytimes/android/external/store/base/impl/MemoryPolicy.java

```diff
@@ -44,16 +51,28 @@ public long getMaxSize() {
     }
 
     public boolean isDefaultPolicy() {
-        return expireAfter == -1;
+        return expireAfterWrite == DEFAULT_POLICY;
     }
 
     public static class MemoryPolicyBuilder {
-        private long expireAfter = -1;
+        private long expireAfterWrite = -1;
+        private long expireAfterAccess = -1;
         private TimeUnit expireAfterTimeUnit = TimeUnit.SECONDS;
         private long maxSize = 1;
 
-        public MemoryPolicyBuilder setExpireAfter(long expireAfter) {
-            this.expireAfter = expireAfter;
+        public MemoryPolicyBuilder setExpireAfterWrite(long expireAfterWrite) {
+            if (expireAfterAccess != -1) {
```

Comment:
Use constant here too, there should be no references to - 1 besides the constant